tests/test_time_machine.py

```diff
@@ -558,6 +558,31 @@ class Something:
     assert excinfo.value.args == ("Can only decorate unittest.TestCase subclasses.",)
 
 
+@time_machine.travel(EPOCH)
+class BaseClassForInheritanceTest(TestCase):
+    prop: bool
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.setUpTestData()
+
+    @classmethod
+    def setUpTestData(cls) -> None:
+        cls.prop = True
+
+
+class ChildClassForInheritanceTest(BaseClassForInheritanceTest):
+    @classmethod
+    def setUpTestData(cls) -> None:
+        cls.prop = False
+
+    @pytest.mark.skip()
```

Comment:
At first I was like “this test case doesn’t fail”, but then I spotted the skip. FYI in future, if you’re adding a failing test case, don’t skip it! Either leave it failing or use xfail: https://docs.pytest.org/en/6.2.x/skipping.html#xfail-mark-test-functions-as-expected-to-fail